lib/src/links/http/fallback_http_config.dart

```diff
@@ -0,0 +1,19 @@
+Map<String, bool> defaultHttpOptions = {
```

Comment:
Coding around dynamic seems error prone to me. What do you think about using mutable objects with a basic builder pattern?:

```dart
class HttpOptions {  
  bool includeQuery = true;
  bool includeExtensions = false;
}
// user code:
HttpOptions options = HttpOptions()
  ..includeQuery = true
  ..includeExtensions = true;
```

For objects over the wire, Json serialization can be achieved like in the subscriptions PoC.

Reply:
@cal-pratt Nice suggestion